addons/stock/wizard/product_replenish.py

```diff
@@ -22,11 +22,14 @@ class ProductReplenish(models.TransientModel):
     warehouse_id = fields.Many2one(
         'stock.warehouse', string='Warehouse', required=True,
         domain="[('company_id', '=', company_id)]")
-    route_ids = fields.Many2many(
-        'stock.location.route', string='Preferred Routes',
-        help="Apply specific route(s) for the replenishment instead of product's default routes.",
+    route_id = fields.Many2one(
+        'stock.location.route', string='Preferred Route',
+        help="Apply specific route for the replenishment instead of product's default routes.",
         domain="['|', ('company_id', '=', False), ('company_id', '=', company_id)]")
     company_id = fields.Many2one('res.company')
+    forecasted_quantity = fields.Float(string="Forecasted Quantity")
```

Comment:
It should be better to have a compute who depends of the `warehouse_id` and the `product_id`, because for now, the forecast quantity isn't updated when the `product_id` is changed (when a product have variants)